_commands/basics/man.md

```diff
@@ -3,22 +3,31 @@
 
 man
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`man` provides *manual pages* for terminal commands
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ man git
+q // to quit
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 

```

Comment:
Show how to lookup man pages in a specific section. For example, with the output from your `man -f` command below, `git(1)` is an output, explaining what this `(1)` means, and how to find commands with different numbers in that place. For example: `man 1  printf` and `man 3 printf` do different things.